packages/express/src/middlewares/requestId.ts

```diff
@@ -0,0 +1,12 @@
+import { v4 } from 'uuid';
+import { RequestHandler } from 'express';
```

Comment:
Should we compromise backward compatibility and enforce the use of randomUUID that became available only on node version 14.17 and higher?